dev-tools.sh

```diff
@@ -33,6 +33,6 @@
 # stable version.
 #
 # Uncomment and set the Dev-Tools's commit value and commit this change.
-# export GH_COMMIT=COMMIT_SHA
```

Comment:
Do not merge. For testing only until a release of `dev-tools`

Reply:
we will merge it. and delete it after all of client sites get updated to 8.7+
@anthony-malkoun